rabbitmq/tests/test_openmetrics.py

```diff
@@ -142,13 +143,60 @@ def test_unaggregated_endpoint(endpoint, fixture_file, expected_metrics, aggrega
     _common_assertions(aggregator)
 
 
+@pytest.mark.parametrize(
+    'endpoint, fixture_file, expected_metrics',
+    [
+        pytest.param(
+            'detailed?family=queue_delivery_metrics',
+            "detailed-queue_delivery_metrics.txt",
+            {
+                'rabbitmq.queue.get.ack.count',
+                'rabbitmq.queue.get.count',
+                'rabbitmq.queue.messages.delivered.ack.count',
+                'rabbitmq.queue.messages.delivered.count',
+                'rabbitmq.queue.messages.redelivered.count',
+                'rabbitmq.queue.messages.acked.count',
+                'rabbitmq.queue.get.empty.count',
+            },
```

Comment:
```suggestion
	RABBITMQ_4_0_ADDED,
```
I think this would be better and more readable.
The suggestion requires adding the necessary import `from .metrics` with `RABBITMQ_4_0_ADDED`